<?xml version="1.0"?>

<launch>
 


    <node pkg="final_assignment" type="robot_interface"    name="robor_interface" output="screen" />
    <node pkg="final_assignment" type="Rand_Server" name="Rand_Server"/>
    <node pkg="final_assignment" type="wall_follow_service_m.py"  name="wall_follower" />

	

</launch>
